packages/presentation/lib/screen/home_bloc.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:domain/use_case/palindrome_use_case.dart';
+import 'package:presentation/base/base_bloc.dart';
+import 'package:presentation/screen/home_data.dart';
+
+abstract class HomeBloc extends Bloc {
+  factory HomeBloc(
+    PalindromeUseCaseImpl palindromeUseCaseImpl,
+  ) =>
+      _HomeBloc(palindromeUseCaseImpl);
+
+  void checkPalindrome();
+
+  void setString(String text);
```

Comment:
for the future - naming setString says about nothing. think over renaming it

Reply:
renamed